### ISSUE TITLE
how to implement custom player controls? 

### ISSUE THREAD

**Issue by faceyspacey** [inProgress-team/react-native-youtube#44]
I'd like to add this feature, or perhaps it's something I do on top of the component. I'd assume you hide the controls, set it to inline. and then use the `onProgress` handler and `seekTo` method from your custom control bar. I don't see a `play` or `pause` method, but I assume I can access it at something like `NativeModules.YouTubeManager.play/pause()`?

It seems I could mostly implement this in client code, but I could also provide a pull request that makes it a built-in feature. I imagine adding a prop `controlBar` which allows developers to supply a custom control bar stylized however they want. 

Any tips would be much appreciated.

**Comment by faceyspacey** [comment]
Also, judging by this: 

https://github.com/paramaggarwal/react-native-youtube/blob/dd87b1e7e2478d5fce656ed17e2fa9e211961316/RCTYouTubeManager.m#L61

it seems that only `seekTo` has been implemented.

Not any of the other methods listed here:

https://developers.google.com/youtube/iframe_api_reference#Queueing_Functions

I assume, for example, implementing `playVideo` would be just:

```
RCT_EXPORT_METHOD(playVideo:(nonnull NSNumber *)reactTag)
{
  [self.bridge.uiManager addUIBlock:^(__unused RCTUIManager *uiManager, NSDictionary<NSNumber *, UIView *> *viewRegistry) {
         RCTYouTube *youtube = viewRegistry[reactTag];

         if ([youtube isKindOfClass:[RCTYouTube class]]) {
             [youtube playVideo];
         } else {
             RCTLogError(@"Cannot Play Video");
         }
     }];
}

@end
```

Let me know if what I envision here is not possible for some reason,

**Comment by faceyspacey** [comment]
...i guess `play={this.state.isPlaying}` is the way to go for pause/play as you have in your example:

https://github.com/paramaggarwal/react-native-youtube/blob/master/Example/index.ios.js


...that all said, I've always felt like a shared interface should be built for Youtube's iframe player and the HTML5 video player or the react native equivalent: https://github.com/brentvatne/react-native-video

Basically, the following custom control bar skin from the *react-native-video* package should be easy to apply to the youtube player as copy/paste. It actually seems easier than ever here in react native--it's just about aliasing a few methods such, e.g. `play` vs. `playVideo` etc. 


https://github.com/brentvatne/react-native-video/blob/master/Examples/VideoPlayer/index.ios.js

**Comment by Mokto** [comment]
Hi, sorry but play parameter is enough and more react-compliant.

;)
